src/compiler/resolutionCache.ts

```diff
@@ -846,6 +825,12 @@ export function createResolutionCache(resolutionHost: ResolutionCacheHost, rootD
         }
     }
 
+    function isInRootPathOrCanWatchDirectoryOrFile(locationToWatch: string) {
+        const path = resolutionHost.toPath(locationToWatch);
+        return isInDirectoryPath(rootPath, path) ||
+            canWatchDirectoryOrFile(resolutionHost.toPath(path));

```

Comment:
Isn't this the same thing as writing `resolutionHost.toPath(resolutionHost.toPath(locationToWatch))`?